barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -271,20 +292,31 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
      * polynomial, extract the value at row_idx. Use these values to create a univariate polynomial, and then extend
      * (i.e., compute additional evaluations at adjacent domain values) as needed.
      * @todo TODO(https://github.com/AztecProtocol/barretenberg/issues/751) Optimize memory
+     *
+     *
      */
-    void extend_univariates(ExtendedUnivariates& extended_univariates,
+
```

Comment:
Can you please explain the addition of the template parameter, also I think the additional Type in name is not necessarily aiding clarity

Reply:
Just used skip_count